app/javascript/styles/_application-custom.scss

```diff
@@ -0,0 +1,21 @@
+.a,
+.govuk-link {
+  color: #1d70b8;
+  &__link:link {
+    color: #003078;
+  }
+  &__visited {
+    color: #4c2c92;
+  }
+  &__link:focus {
+    color: #0b0c0c;
+  }
+  &__link:hover {
+    color: #003078;
+  }
+}
```

Comment:
Not sure you need these styles as they exist in `govuk-frontend` CSS. If the intention is to automatically style any link correctly, there is a setting that can be added to our application SCSS file:

`$govuk-global-styles: true;`

This will automatically style links as if they had the `govuk-link` class (and paragraphs as if they had the `govuk-body` class). Opinions on this setting are mixed; possibly bit of a blunt instrument, but it works (we use this setting in the prototype).

Reply:
Oh wait… where are `.govuk-link__link` and `.govuk-link__visited` coming from?

Reply:
Was taking it directly from browser inspector on the prototype, but have replaced it with the global setting now.

Reply:
Cool. I don’t think you need this separate stylesheet; this setting can go at the top of `app/javascript/styles/application.scss`

Reply:
Done